deployment/production/deploy.sh

```
@@ -0,0 +1,52 @@
+docker compose build
+
+echo "Starting containers..."
+docker compose restart nginx
+docker compose up -d
+
+# Get deployment status
+if [ $? -eq 0 ]; then
+  DEPLOY_STATUS="success"
+  MESSAGE="✅ **$PROJECT_NAME** has been successfully deployed to **$ENVIRONMENT** (backend.arkadtlth.se).\nCommit: \`$LAST_COMMIT_HASH\`\nMessage: \`$COMMIT_MESSAGE\`"
+else
+  DEPLOY_STATUS="failure"
```

Shellcheck (0.11.0)

[warning] DEPLOY_STATUS appears unused. Verify use (or export if used externally).

(SC2034)